E1a-ARSE/ALSE/Dataset/README.md

```diff
@@ -0,0 +1,50 @@
+## Variations
+The Following occurence variations need to be checked and certified in relation to the targeted entities and concept templates:
+
+- (Entity_01) IfcAnnotation/SUPERELEVATIONEVENT - *Pset_SuperElevation needs all variants of Side and TransitionSuperelevation tested. Furthermore, values for Superelevation shall include both positive and negative values*
+
+- (Entity_02) IfcAnnotation/WIDTHEVENT - *Pset_Width needs all variants of Side and TransitionWidth tested.*
+
+## Model Dataset
+This test case utilises the attached dataset documented by the following drawings and data schedule.
+
+"Synopsis" (unique entities for this test in **bold**):
+
+- IfcProject
+
+  - IfcGeometricRepresentationContext
+
+    - IfcMapConversion
+      - IfcProjectedCRS
+
+  - IfcSite
+
+    - IfcAlignment
+      - Should have straights, clothoid/circular arc to the left and to the right
+      - Vertical is not really needed but is probably part of the included E1-ALRF
+
+    - IfcRoad
+      - IfcFacilityPart/ROADSEGMENT
+        - **IfcFacilityPart/CARRIAGEWAY**
+          - **IfcAnnotation/SUPERELEVATIONEVENT (one for every start of alignment element)**
+            - **Pset_Superelevation (left side)**
+            - **IfcLinearPlacement**
+          - **IfcAnnotation/SUPERELEVATIONEVENT (one for every start of alignment element)**
+            - **Pset_Superelevation (right side)**
+            - **IfcLinearPlacement**
+          - **IfcAnnotation/WIDTHEVENT (one at start, add a narrowing of the road somewhere along the alignment)**
```

Comment:
Not sure I understand the indentations. Are we supposed to do that?

Reply:
No, I wouldn't say that. This was just for me to sketch a possible structure for the imagined file to export. Indentations are just to i achieve readability (at least for me), similar to Jon's Tree View app.